frontend/src/components/common/pagination/pagination.tsx

```diff
@@ -40,7 +40,7 @@ const Pagination: FC<Props> = ({
         <button
           className={styles.next}
           onClick={handleNextPageChange}
-          disabled={currentPage === totalPages}
+          disabled={currentPage === totalPages || totalPages === 0}
```

Comment:
```suggestion
          disabled={isNextBtnDisabled}
```
add a constant for `0` at the top of the file